File: projects_cpp/A_Demo/Human.cs
using System;

class Human
{
    private int _id;
    public string Name { get; set; }
    public string Occupation { get; set; }
    public string Age { get; set; }

    public Human(int id, string name, string occupation, int age)
    {
        this._id = id;
        name = name;
        occupation = occupation;
        age = age;
    }
    
    public void Speak(string message)
    {
        Console.WriteLine(message);
    }
}

File: projects_cpp/A_Demo/Human.cpp
#include <iostream>

class Human
{
    private:
        int id;
    
    public:
        std::string name;
        std::string occupation;
        int age;

    Human(std::string name, std::string occupation, int id, int age)
    {
        this->name = name;
        this->occupation = occupation;
        this->age = age;
        this->id = id;
    }

    void speak(std::string message);

};

void Human::speak(std::string message) 
{
    std::cout << message << '\n';
}

int main()
{
    return 0;
}